tools/bootstrap/README.md

```diff
@@ -25,6 +25,9 @@ Install-BoxstarterPackage https://raw.githubusercontent.com/nodejs/node/master/t
 
 Entire installation will take up about 10 GB of disk space.
 
+### Why install two different versions of Python?
+Python 2 will reach its _end-of-life_ at the end of 2019. Afterwards, the interpreter will not get updates -- no bugfixes, no security fixes, nothing. In the interim, the Python ecosystem is abandoning 2.7 support. https://python3statement.org/  In order to remain safe and current the Node community is transitioning its Python code to Python 3.  Having both versions of Python in this bootstrap will allow developers and end users to test, benchmark, and debug Node running on both platforms to ensure a smooth and complete transition before the yearend deadline.
```

Comment:
Can you break these lines with 80 chars limit per line? Also, I prefer Node.js over Node.